teams/miri.toml

```diff
@@ -8,6 +8,9 @@ alumni = ["solson"]
 
 [permissions]
 bors.miri.review = true
+bors.rust.try = true
+crater = true
```

Comment:
If crater runs have never been needed for miri, and they're relatively rare as is, I think it makes sense to remove this for now.